File: 351001/shevchenkoAD/DistComp1/Domain/Entities/Issue.cs
using Domain.Abstractions;

namespace Domain.Entities;

public class Issue : BaseEntity {
    public long AuthorId { get; set; }
    public string Title { get; set; } = "";
    public string Content { get; set; } = "";
    public DateTime Created { get; set; }
    public DateTime Modified { get; set; }
    public List<long> LabelIds { get; set; } = new();
}

File: 351001/Perova/app/models/issue.py
from dataclasses import dataclass, field
from datetime import datetime

from app.models.base import BaseEntity


@dataclass
class Issue(BaseEntity):
    userId: int = 0
    title: str = ""
    content: str = ""
    created: datetime | None = None
    modified: datetime | None = None
    stickerIds: list[int] = field(default_factory=list)
